style.css

```diff
@@ -46,3 +46,9 @@ th {
 .add-book {
   width: 50%;
 }
+
+.add-book-title,
+.text-center {
+  font-weight: bold;
+  margin-bottom: 2%;
+}
```

Comment:
While you're free to use styles of your choice (colors, fonts, backgrounds, etc.), we need to ensure that the general layout for the project is followed as much as possible.
![screenshot](https://user-images.githubusercontent.com/26856491/125658789-b252fc46-a0a3-4570-8d95-375e93aa311f.png)

Looking at the image above, we observe the following changes required:
- [ ] For each book displayed in the layout, the text structure is kind of different from your structure. For instance, we have:
- "Title" by Author, meaning that you should put your title in the double quote, followed by the `by` word, then the Author.
- Also, it looks like we have just 2 columns in the table, whereas you're having 3 columns.
- If the structure highlighted above is followed, then you can remove the column title (`Book`, `Title`, `Action`) on your table

- [ ] For the `Add Book` button text, I think you should change the text to `Add` instead of the `Submit Query` we currently have.

Reply:
Thank you for the review. I have now changed the style according to the requirements.